app/templates/admin/userManagement.html

```diff
@@ -241,6 +248,37 @@ <h5 class="modal-title">Update Program Info</h5>
       </div>
     </div>
   </div>
-  
+
+  <!-- Program Management Modal -->
+  <div class="modal fade" id="editProgramManagers">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="programNameHeader"> Edit Program Managers </h5>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
+          <a id="programPlaceholder" class="d-none" data-programid=""></a>
+          <div class="modal-body">
+            <div id="parentManager">
+              {{createInputsButtons("searchProgramManagersInput", "Add Manager")}}<br>
+            </div>
+            <table class="table mb-0">
+            <thead>
+              <tr>
+                <th colspan="2" scope="col">Program Managers</th>
+              </tr>
+            </thead>
+            <tbody id="programManagersTable">
+              
+            </tbody>
+          </table>
+        <p id="noManagersText" class="text-center mt-2">This program has no assigned managers.</p>
```

Comment:
Make sure this doesn't show when you add a program manager to a program without them.